src/Panel/Panel.js

```diff
@@ -148,6 +149,7 @@ export default class Panel extends React.Component {
       header ?
         <PanelHeader
           title={header.hideTitleWhenExpanded && isExpanded ? undefined : header.title}
+          size={header.size || HeaderSize('small')}
```

Comment:
Ok, choose by yourself in which `defaultProps` to put it, but I think this default should be defined in `defaultProps` as we're doing for the other props (I personally think it makes the component more readable as there is a standard place just below `@props` to see every eventual default)